docs/howto/operate/move-hub.md

```diff
@@ -23,29 +23,67 @@ home directory storage!
 
 Primarily used with GKE right now.
 
-1. In the target NFS server, create a new ssh key-pair, with
+1. SSH into **both** the target server and source server and then enable the `ubuntu` user on **both** servers with `sudo su ubuntu`.
```

Comment:
Might be a stupid question, but do our clusters all have an `ubuntu` user on them?

Reply:
It's not our clusters but a VM that is either acting as an NFS server or a VM that has some kind of storage mounted (GCP Filestore, Azure Filestore, etc).

I _think_ most linux distros ship with an `ubuntu` user by default - I didn't have to `sudo useradd ubuntu` - but I'm not 100% sure.